### ISSUE TITLE
Add a Dynamic Gas Size

### ISSUE THREAD

**Issue by oojr** [etherdelta/etherdelta.github.io#309]
when the network is congested the default 4 gwei will not work, the gas size should be more dynamic

**Comment by coinaisseur** [comment]
I agree with this request. Currently the average gas price is 53 GWEI according to ethstats.net, and the default gas price should be similar to this average, otherwise pretty much all users will be left with unconfirmed transactions, either because they don't know how to change the default price, or forget to do so.
